boutiques_descriptors/scrnabox_1.52.5.json

```diff
@@ -0,0 +1,1509 @@
+            "description": "If you want to perform sample-based DGE with all cells",
+            "optional": true
+        },
+        {
+            "name": "If you want to perform sample-based DGE on each cell type group",
+            "command-line-flag": "--par_run_sample_based_celltype_groups",
+            "value-key": "[PAR_RUN_SAMPLE_BASED_CELLTYPE_GROUPS]",
+            "type": "Flag",
+            "id": "par_run_sample_based_celltype_groups",
+            "description": "If you want to perform sample-based DGE on each cell type group",
+            "optional": true
+        },
+        {
+            "name": "Which statistical method to use when computing DGE using individual cells as replicates",
+            "command-line-flag": "--par_statistical_method",
+            "value-key": "[PAR_STATISTICAL_METHOD]",
+            "type": "String",
+            "id": "par_statistical_method",
+            "description": "Which statistical method to use when computing DGE using individual cells as replicates",
+            "optional": true
+        },
+        {
+            "name": "To perform cell-based DGE using all cells",
+            "command-line-flag": "--step8_contrast_cell_based_all_cells",
+            "value-key": "[STEP8_CONTRAST_CELL_BASED_ALL_CELLS]",
+            "type": "File",
+            "id": "step8_contrast_cell_based_all_cells",
+            "description": "To perform cell-based DGE using all cells",
+            "optional": true
+        },
+        {
+            "name": "To perform cell-based DGE using cell type groups",
+            "command-line-flag": "--step8_contrast_cell_based_celltype_groups",
+            "value-key": "[STEP8_CONTRAST_CELL_BASED_CELLTYPE_GROUPS]",
+            "type": "File",
+            "id": "step8_contrast_cell_based_celltype_groups",
+            "description": "To perform cell-based DGE using cell type groups",
+            "optional": true
+        },
+        {
+            "name": "To perform sample-based DGE using all cells",
+            "command-line-flag": "--step8_contrast_sample_based_all_cells",
+            "value-key": "[STEP8_CONTRAST_SAMPLE_BASED_ALL_CELLS]",
+            "type": "File",
+            "id": "step8_contrast_sample_based_all_cells",
+            "description": "To perform sample-based DGE using all cells",
+            "optional": true
+        },
+        {
+            "name": "To perform sample-based DGE using all cells",
+            "command-line-flag": "--step8_contrast_sample_based_celltype_groups",
+            "value-key": "[STEP8_CONTRAST_SAMPLE_BASED_CELLTYPE_GROUPS]",
+            "type": "File",
+            "id": "step8_contrast_sample_based_celltype_groups",
+            "description": "To perform sample-based DGE using all cells",
+            "optional": true
+        }
+    ],
+    "output-files": [
+        {
+            "id": "outdir",
+            "name": "The output directory",
+            "path-template": "scRNAbox_dir",
```

Comment:
I don't understand this. This is an option that the user can provide, yet the path-template is a fixed string?